usage/app/lib/UsageRecorder.scala

```diff
@@ -14,8 +14,9 @@ class UsageRecorder(usageMetrics: UsageMetrics, usageTable: UsageTable, usageStr
   val liveUsageStream: Observable[UsageGroup] = usageStream.liveObservable.merge(usageSubject)
 
   val subscriber = Subscriber((_:Any) => Logger.debug(s"Sent Usage Notification"))
-  def subscribeToPreview  = previewObservable.subscribe(subscriber)
-  def subscribeToLive  = liveObservable.subscribe(subscriber)
+
+  var subscribeToPreview: Option[Subscription] = None
```

Comment:
Why do these need to be `var`?

Reply:
We assign to them in the `start` method so we can call `unsubscribe` in the `stop` method.

I don't think there's a reason why we wouldn't just start them when they are constructed but the code prior to the upgrade kicked them off manually in `Global.scala` so I was keeping to the spirit of that code